plugins/module_utils/turbo/lookup.py

```diff
@@ -0,0 +1,130 @@
+import json
```

Comment:
This is a a but of duplicated code in this one. Can we move that in a shared module_utils file?

Reply:
I will do that